index.html

```diff
@@ -28,13 +33,150 @@ <h1>I'm Chime Glad to see you!</h1>
+            <button type="button" class="btn">html</button>
+            <button type="button" class="btn">css</button>
+            <button type="button" class="btn">JavaScript</button>
+            </div>
+          </div>
+            <button type="button" class="big-btn">See Project</button>
+
+        </div>
+        <div class="projects">
+            <div class="header">
+                <img src="media/Snapshoot2.svg" alt="picture">
+                </div>
+                <div class="content">
+                    <h1>Topic</h1>
+                    <div class="data">
+                        <p>CANOPY</p>
+                        <p  class="para">Back-End dev</p>
+                        <p>2022</p>
+                    </div>
+                <p>A daily selection of personalized reads; no accounts or sign-ups required</p>
+                
+                <div class="btns">    
+                <button type="button" class="btn">html</button>
+                <button type="button" class="btn">css</button>
+                <button type="button" class="btn">JavaScript</button>
+                </div>
+              </div>
+                <button type="button" class="big-btn">See Project</button>
+            </div>
+        <div class="projects">
+            <div class="header">
+                <img src="media/Snapshoot3.svg" alt="picture">
+                </div>
+                <div class="content">
+                    <h1>Topic</h1>
+                    <div class="data">
+                        <p>CANOPY</p>
+                        <p class="para">Back-End dev</p>
+                        <p>2022</p>
+                    </div>
+                <p>A daily selection of personalized reads; no accounts or sign-ups required</p>
+                
+                <div class="btns">    
+                <button type="button" class="btn">html</button>
+                <button type="button" class="btn">css</button>
+                <button type="button" class="btn">JavaScript</button>
+                </div>
+              </div>
+                <button type="button" class="big-btn">See Project</button>
+            </div>
+        <div class="projects">
+            <div class="header">
+                <img src="media/Snapshoot4.svg" alt="picture">
+                </div>
+                <div class="content">
+                    <h1>Topic</h1>
+                <div class="data">
+                    <p>CANOPY</p>
+                    <p class="para">Back-End dev</p>
+                    <p>2022</p>
+                </div>
+                <p>A daily selection of personalized reads; no accounts or sign-ups required</p>
+                
+                <div class="btns">    
+                <button type="button" class="btn">html</button>
+                <button type="button" class="btn">css</button>
+                <button type="button" class="btn">JavaScript</button>
+                </div>
```

Comment:
- [ ] As you can see in screenshots below the list of project technologies (e.g., "JavasScript", "HTML" and "CSS") should be a list `ul` and each of them is a `li`

```
<ul>
  <li>..</li>
  <li>..</li>
  <li>..</li>
</ul>
```
![Capture](https://user-images.githubusercontent.com/80895497/153720337-6b5779a9-fdae-4ae7-b31e-09f0d8c27b96.PNG)